tools/ci/README.md

```diff
@@ -1,3 +1,26 @@
 # CI Tools
 
 This folder contains miscellaneous utilities for CI.
+
+# Docker
+
+## Building
+
+A custom docker image requires a certain number of directories to be present
+from a desired _edition_. The root directory of the docker build context can be
+provided in the `docker build` command:
+
+```bash
```

Comment:
> certain number of directories to be present

What's the command to invoke before invoking `docker build` to get the _"certain number of directories"_?

Reply:
It's this argument `built-distribution/enso-engine-$VERSION-linux-amd64/enso-$VERSION`